mockld/streaming_service.go

```diff
@@ -80,6 +84,16 @@ func NewStreamingService(
 	switch sdkKind {
 	case ServerSideSDK:
 		router.HandleFunc(StreamingPathServerSide, streamHandler).Methods("GET")
+	case RokuSDK:
+		config := crypto.CryptoConfig{
+			AuthKey:   "YWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4eXowMTIzNDU=",
+			CipherKey: "YWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4eXowMTIzNDU=",
```

Comment:
Wasn't sure what we wanted to do here. It just needs to be 2 32-character strings, and then base64 encoded. These are a-z0-5. :shrug: